src/xdocs/checks/coding/unusedlocalvariable.xml.template

```diff
@@ -17,21 +17,46 @@
         </p>
       </subsection>
 
+      <subsection name="Properties" id="Properties">
+        <div class="wrapper">
+          <macro name="properties">
+            <param name="modulePath"
+                   value="src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/UnusedLocalVariableCheck.java"/>
+          </macro>
+        </div>
+      </subsection>
+
       <subsection name="Examples" id="Examples">
         <p id="Example1-config">
           To configure the check:
         </p>
         <macro name="example">
           <param name="path"
-                 value="resources/com/puppycrawl/tools/checkstyle/checks/coding/unusedlocalvariable/Example1.java"/>
+                 value="resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/unusedlocalvariable/Example1.java"/>
           <param name="type" value="config"/>
         </macro>
         <p id="Example1-code">
           Example:
         </p>
         <macro name="example">
           <param name="path"
-                 value="resources/com/puppycrawl/tools/checkstyle/checks/coding/unusedlocalvariable/Example1.java"/>
+                 value="resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/unusedlocalvariable/Example1.java"/>
+          <param name="type" value="code"/>
+        </macro>
+        <p id="Example2-config">
+           To configure the check to violate underscore variables if they are not used:
```

Comment:
Please stick to language like `Do not place violations on variables named with a single underscore (known as <a href="https://docs.oracle.com/en/java/javase/21/docs/specs/unnamed-jls.html">
 unnamed variables</a> in Java 21+).`, `underscore variables` is nonsense 😂

Reply:
done